src/pages/Home/home.js

```diff
@@ -1,7 +1,43 @@
-export const Home = () => {
-  return (
-    <>
-      New Home
-    </>
-  )
-}
\ No newline at end of file
+import axios from "axios";
+import React from "react";
+
+export class Home extends React.Component {
+  state = {
+    data: {},
+  };
+  getBooksByGenre = async (subject) => {
+    try {
+      const { data } = await axios(
+        `https://www.googleapis.com/books/v1/volumes?q=subject:${subject}&maxResults=40&key=${process.env.REACT_APP_API_KEY}`
+      );
+      this.setState({ data: data.items });
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  componentDidMount() {
+    this.getBooksByGenre("horror");
+  }
+
+  MappedData = () => {
+    const mappedData = this.state.data.map((object) => {
+      return (
+        <div>
+          {object.volumeInfo.title}
```

Comment:
object is a bit vague, if I did not know you are talking about books I would be confused. Rename it to `book`

Reply:
I made the change. Thanks!